### PR TITLE
Skipping boot test for Alma Linux

### PR DESCRIPTION
**Pull request by KarnVadaliya**

### CODE REVIEW

File: imagetest/test_suites/image_boot/setup.go [hunk 1]
```diff
@@ -38,6 +38,11 @@ func TestSetup(t *imagetest.TestWorkflow) error {
 		return nil
 	}
 
+	if strings.Contains(t.Image, "almalinux") {
+		// secure boot is not supported on Alma Linux
```
Comment (zmarano) [inline imagetest/test_suites/image_boot/setup.go:42]: Secure boot is supported both on Rocky Linux and AlmaLinux.... but is this test even about secure boot?

Comment (KarnVadaliya) [reply imagetest/test_suites/image_boot/setup.go:42]: Both the test (secure boot and boot time) runs on the same VM so skipping for both. Should I run them on separate VMs ?

Comment (zmarano) [reply imagetest/test_suites/image_boot/setup.go:42]: Oh I see.... we don't need to skip in either case- just add almalinux to the supported boot time tests.

Comment (KarnVadaliya) [reply imagetest/test_suites/image_boot/setup.go:42]: Also, skipping secure boot for rocky-linux-8 was already there before. As of now it is supported ?

Comment (zmarano) [reply imagetest/test_suites/image_boot/setup.go:42]: Yeah- it is supported so we should actually remove that skip (I thought I did already).

**Pull request closed without merging**